plugins/scaffolder-backend-module-cookiecutter/src/actions/index.ts

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright 2021 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+export { createFetchCookiecutterAction } from './fetch';
```

Comment:
Shouldn't a reexport of an index be done as a wildcard export? https://backstage.io/docs/architecture-decisions/adrs-adr004
```suggestion
export { * } from './fetch';
```